app/helpers/application_helper.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
-  def full_title(page_title = '')
-    base_title = I18n.t 'helpers.application.base_title'
+
+  # Returns the full title on a per-page basis
+  def full_title page_title = ""
+    base_title = I18n.t "helpers.application.base_title"
     if page_title.empty?
```

Comment:
ngoài dùng `if ... else ... end` thì còn cách nào khác tương đương không em?

Reply:
e nghĩ có thể dùng lệnh switch case

Reply:
còn cách nào khác nữa không em, trường hợp này mà dùng `switch ... case` thì có hợp lý ko?

Reply:
e chưa nghĩ ra cách nào khác nữa chị ạ

Reply:
em tìm hiểu thêm cấu trúc điều khiển rẽ nhánh:
`unless ... else`
`until`
Toán tử `... ? ... : ...`